packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -79,6 +79,15 @@
 TransactionType = TypeVar("TransactionType")
 
 
+def get_name(prop: Any) -> str:
+    """Get the name of a property."""
+    if not (isinstance(prop, property) and hasattr(prop, "fget")):
+        raise ValueError(f"{prop} is not a property")
+    if prop.fget is None:
+        raise ValueError(f"{prop.fget} is None")
+    return prop.fget.__name__
```

Comment:
We can also set the `payload_attribute` automatically in the `AbstractRound` - only if not set so we keep the backward compatibility.

Reply:
Correct. But only if we assume that the payload only has a single property. This assumption is not uniformly true yet (see eg RandomnessRound), but I think we should force that as part of `0.6.0`.